definitions/palabritas.md

```diff
@@ -0,0 +1,243 @@
+        * (50%) I leave two dollars to the artist.
+          The musician smiles and nods thanking you.
+        * I nod when the musician looks my way, to show I really enjoy the music.
+      [...]
+```
+
+If a choice doesn't have probability set, the probability of it showing up is 100%.
+
+In this case there is a 50% chance the user will see the option to leave two dollars to the musician. So half of the times he'll support the artist economically, and half just nod.
+
+#### Options bucket
+
+If all the options in an indentation level are probabilistic, then a bucket is created.
+
+This means that only one of those options will show up, respecting the probability rules.
+
+```cuentitos
+    (500) Wandering the market, you stumble upon a hidden alley adorned with vibrant street art. 
+      Each colorful mural tells a different story, capturing your imagination and sparking your creativity. 
+        * (50%) I grab my phone and take pictures of the murals.
+        * (50%) I keep walking, even if the murals look good, the darkness of the alley is unsettling.
+```
+
+In this case, only one of the options will show up, each one with a 50% chance.
+
+## Reacting to State
+
+Up until this point, we've not used the game state to setup conditions. So let's do that now. We'll start writing from the root indentation level again. This texts will show up when the previous section is navigated until there is no more text to show. It will automatically backtrack to the root if no texts display in any indentation level.
+
+```cuentitos
+Feeling mentally and physically exhausted from the day's adventures, you decide it's time to head back to your hotel.
+As you enter the peaceful sanctuary of your room, you take a deep breath, relieved to have a quiet space where you can recharge and prepare for the challenges ahead.
+
+The sun shines bright through the window.
+  req time_of_day !night
+The moonlight gives the room a peaceful tone.
+  req time_of_day night
+```
+
+In this case, we're putting conditions to these two lines of text. They will only show up if the `time_of_day` variable value satisfies the conditions. The first one will show if time of day is not `night`, the second one, will show when time of day is `night`.
+
+Check the `Configuration` section to learn about variables.
+
+In this case, the requirements are mutually exclusive, only one will show at a time. But that's not a requirement. You could have multiple lines with different requirements and have them show one after the other if the conditions are satisfied.
+
+```cuentitos
+The sun shines bright through the window.
+  req time_of_day !night
+The moonlight gives the room a peaceful tone.
+  req time_of_day night
+You start to think about all the stuff you need to do tomorrow.
+  req time_of_day_night
+That makes you feel overwhelmed.
+  req time_of_day_night
```

Comment:
I think this was meant to be `req time_of_day night` instead of `req time_of_day_night` (using a space instead of an underscore between day and night)

Reply:
Yep